pkgs/development/python-modules/edge-tts/default.nix

```
@@ -0,0 +1,42 @@
+{
+  fetchFromGitHub,
+  lib,
+  buildPythonPackage,
+  # Dependencies
+  aiohttp,
+  certifi,
+  srt,
+  tabulate,
+  typing-extensions,
+}:
+buildPythonPackage rec {
+  pname = "edge-tts";
+  version = "7.0.0";
+  src = fetchFromGitHub {
+    owner = "rany2";
+    repo = "edge-tts";
+    tag = version;
+    hash = "sha256-TwBSEePVFYTvdueunTrFtZd96tp2P/he7+1cmIk2dy8=";
+  };
+
+  propagatedBuildInputs = [
```

```suggestion
  dependencies = [
```
You may also need `build-system = [ setuptools ];`